servers/credentialagent/src/controllers/saasoperator/groups/_id/controller.js

```diff
@@ -40,13 +35,8 @@ const controller = async (fastify) => {
     '/',
     {
       schema: fastify.autoSchema({
-        params: {
-          type: 'object',
-          properties: { id: { type: 'string', minLength: 1 } },
-          required: ['id'],
-        },
```

Comment:
why removed? these are all in autoSchema already?

Reply:
Was already handled in autohooks